juspay/hyperswitch
# feat(payouts): add payout types in euclid crate

## Type of Change



## Description

### Additional Changes





## Motivation and Context



## How did you test it?

_**Note -** DM for postman collection_

Locally, using HyperSwitch's dashboard.
PR for changes - https://github.com/juspay/hyperswitch-control-center/pull/460

Steps for testing
- Use the appropriate version of dashboard for testing (source code [here](https://github.com/juspay/hyperswitch-control-center/tree/payout_routing))
- Login to dashboard
- Configure two payout connectors
- Head to **Workflow -> Payout Routing** section
- Configure Rule based config (payment_method - card -> Adyen; payment_method - bank -> Wise)
- Activate this config
- Head to postman collection and run it

Expectations
- Card payouts should be routed through Adyen
- Bank payouts should be routed through Wise

## Checklist
